app/src/main/java/com/isc/hermes/EmailVerificationActivity.java

```diff
@@ -34,7 +34,7 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_mail_verification);
         validator = Validator.getValidator();
-        validator.obtainVerificationCode();
+        //validator.obtainVerificationCode();
```

Comment:
If you are not using this line, it should be deleted.

Reply:
Done